package.json

```diff
@@ -33,7 +33,7 @@
     "autosize": "^4.0.2",
     "axios": "^0.19.0",
     "axios-extensions": "^3.0.6",
-    "bootstrap-vue": "^2.0.0-rc.25",
```

Comment:
should we use 2.0.1 which is the latest?

Reply:
updated to 2.0.1